Azure-Samples/msdocs-django-postgresql-sample-app
# Pythonic style improvements

## Purpose

This PR improves the style of the Python code in the Django+PG sample to be more Pythonic. I ran isort, ruff, black --skip-string-normalization --line-length 120, and pyupgrade, and then kept the most significant/objectively Pythonic changes. 
Some of the changes do add new lines that would affect line numbers, but they shouldn't affect views.py, the only code that is snippeted in the docs.


## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[X] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Get the code
* Run locally
* Add restaurant, add review, confirm date is correct
